components/VButton.vue

```diff
@@ -8,7 +8,7 @@
                     ? 'text-brand-primary hover:text-white bg-white hover:bg-brand-primary hover:text-white'
                     : null,
                 appearance === 'inverted'
-                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white'
+                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white hover: hover:border-1 border-solid border-sky-500'
```

Comment:
nice!! Great work

Okay couple things: 
1. I would love for the border to be our brand colour. I am thinking you should be able to do that with something like `bg-brand-primary` but I haven't tested that, so it could be wrong
2. You seem to have an extra floating `hover:` thats not attached to anything
3. Technically with tailwind they like when you add `hover:` in front of every property you add, so add a couple more :)